perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -40,8 +40,16 @@ sub disambiguate_location { {
     centre => '51.5585509362304,-0.26781886445231',
     span   => '0.0727325098393763,0.144085171830317',
     bounds => [ 51.52763684136, -0.335577710963202, 51.6003693511994, -0.191492539132886 ],
+    town => 'Brent',
 } }
 
+sub geocoder_munge_results {
+    my $self = shift;
+    my ($result) = @_;
```

Comment:
Is fine, but any reason these two lines aren't `my ($self, $result) = @_;` :)

Reply:
Because I copied from Thamesmead...which was also me!

https://lk.linkedin.com/posts/praveentech_chatgpt-openai-ai-activity-7032275840251883520-vYLN